datalake/backend/endpoints/dataset_versions/create.py

```diff
@@ -0,0 +1,106 @@
+"""Import dataset-version function."""
+import json
+import logging
+import uuid
+from os import environ
+
+import boto3
+from jsonschema import ValidationError, validate  # type: ignore[import]
+from pynamodb.exceptions import DoesNotExist
+
+from ..datasets.model import DatasetModel
+from ..utils import DATASET_TYPES, ENV, JSON_OBJECT, error_response, success_response
+
+stepfunc_client = boto3.client("stepfunctions")
+ssm_client = boto3.client("ssm")
+
+STEP_FUNCTION_ARN_PARAMETER_NAME = f"/{ENV}/StepFuncStateMachineARN"
```

Comment:
I would rename this variable to something more descriptive like `DATASET_VERSION_CREATION_STEP_FUNCTION ...`